guides/release/configuring-ember/debugging.md

```diff
@@ -78,7 +78,8 @@ Ember.ENV.LOG_STACKTRACE_ON_DEPRECATION = true;
 
 ### Implement an Ember.onerror hook to log all errors in production
 
-```javascript
+```javascript {data-filename=app/app.js}
+import Ember from 'ember';
 import fetch from 'fetch';
 
```

Comment:
@nvdk thanks for making this change 😄. What do you think about adding an ellipsis below the imports but before the `onerror` code to demonstrate that this hook could be introduced in a variety of types of files and that there's other code not displayed?
 
```suggestion
// ...
```